packages/react-native-bottom-tabs/src/TabView.tsx

```diff
@@ -201,6 +201,12 @@ interface Props<Route extends BaseRoute> {
    * @platform ios
    */
   renderBottomAccessoryView?: BottomAccessoryViewProps['renderBottomAccessoryView'];
+  /**
+   * The direction of the layout. (iOS only)
+   * @platform ios
+   * @default 'leftToRight'
+   */
+  layoutDirection?: 'leftToRight' | 'rightToLeft';
```

Comment:
We should allow for three options:

`layoutDirection: 'ltr' / 'rtl' / 'locale'`

Using `ltr` and `rtl` as already established patterns. `locale` is the default.

We also need to support Android if we want to roll out this feature.